server/main.lua

```diff
@@ -11,26 +11,60 @@ local availableJobs = {
 }
 
 -- Functions
-
 local function giveStarterItems()
     local Player = QBCore.Functions.GetPlayer(source)
     if not Player then return end
+    local citizenid = Player.PlayerData.citizenid
+    local firstname = Player.PlayerData.charinfo.firstname
+    local lastname = Player.PlayerData.charinfo.lastname
+    local gender = 'Male'
+    local birthdate = Player.PlayerData.charinfo.birthdate
+    local nationality  = Player.PlayerData.charinfo.nationality
+    if not Player.PlayerData.charinfo.gender == 1 then
+        gender = 'Female'
+    else
+        gender = 'Male'
+    end
```

Comment:
Instead of this, you can do
```suggestion
    local gender = Player.PlayerData.charinfo.gender == 1 and 'Male' or 'Female'
    local birthdate = Player.PlayerData.charinfo.birthdate
    local nationality  = Player.PlayerData.charinfo.nationality
```